### DISCUSSION TITLE
Ainneve discussion

### DISCUSSION THREAD

**Discussion by Griatch** [evennia/evennia#2279] [My Evennia Project]
[Ainneve](https://github.com/evennia/ainneve) is a community-driven attempt to make an Evennia 'example game'. The idea is to make a demo game using good practices for people to build from, rather than the more bare-bones approach of Evennia itself. 

This is intended as a community project - the Evennia lead dev has intentionally tried to *not* take an active role in it. 

- [Here's the original announcement](https://groups.google.com/g/evennia/c/48PMDirb7go/m/Z9EAuvXZn7UJ)
- Ainneve is currently pretty stalled. Enthusiastic project-runners are needed, so consider jumping in if you want to take on/help with a project that will be beneficial to everyone! A person willing to take it on would have a lot of freedom in designing/reworking what's currently there. 
- Current code needs to be updated to Python3 and latest Evennia.
- Code should probably make a lot more active use of Evennia contribs.
- To date, Ainneve has implemented a rudimentary character generation and combat based on the [Open Adventure](https://github.com/openadventure/Open-Adventure) tabletop rpg rules. This is not set in stone - we need to use a rule set with a usable license, but but which one is definitely possible to change. 
- The current setup includes a planned city map, but this is open to change, and while we want a fantasy game, the theme and style of it is still up in the air. Overall the game itself need not be bigger than that it showcases all its systems in a good way. 

Feel free to discuss Ainneve stuff in this thread! :smile:

**Comment by Griatch** [comment]
I want to remind people about the Ainneve project. This is intended as a community project (that is, I'm not driving it), but as with all such projects, it requires one or more core persons willing to go the mile and keep working also when others drop off. So if you are interested in taking on an open-source project, this could be something for you! 
It's a great learning opportunity - at this point, most of the code in the Ainneve repo is pretty dated and needs to be updated (or scrapped, if you prefer). While the people originally working on Ainneve made some calls about system etc, there's nothing saying this can't change as long as it fulfills the original goals of the project - to have a useful Evennia demo game.

**Comment by InspectorCaracal** [comment]
I'd actually be interested in spearheading work on this going forward, but I'm a little worried about stepping on toes when it comes to scrapping prior work done. As an experienced open source project manager, do you have advice on how to approach those types of situations?
